.cirrus/clean-orchestrator-cache.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+cd "$ORCHESTRATOR_HOME" || exit 1
```

Comment:
Why not using `set -euo pipefail`?
https://xtranet-sonarsource.atlassian.net/wiki/spaces/Platform/pages/2683109459/Shell+Script+-+Cirrus+CI#Safe-Execution